website/dynamic-blocks/headings.js

```diff
@@ -34,12 +34,13 @@ const Component = ({ heading, size, level, addTableContent, subText, text }) =>
 					</Cell>
 				</Grid>
 			</WestpacHeading>
-
-			<Grid columns={12}>
-				<Cell width={10} left={[1, 1, 2, 2]}>
-					{subText && text && <p css={{ lineHeight: 2 }}>{text}</p>}
-				</Cell>
-			</Grid>
+			{subText && text && (
```

Comment:
what's `subText` and why is it required for the blow below?

Reply:
Kate asked for the subText to be conditionally shown.

Reply:
We we don't display `subText` anywhere below? is that a checkbox somewhere?

Reply:
Yeah subText is a boolean that comes from a checkbox.

Reply:
Makes sense then :) thank you both

Reply:
<img width="340" alt="Screen Shot 2020-04-17 at 1 14 13 pm" src="https://user-images.githubusercontent.com/16239047/79528484-675bfa00-80ad-11ea-84bc-498221afa62c.png">